portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase

```diff
@@ -700,4 +700,23 @@ definition {
                 increaseAttributeNumber = "2");
         }
     }
+    @description = "Story: LRAC-10860 | Automation ID: LRAC-12837 | Test Summary: Cannot deselect required Attributes for Account"
+    @priority = "5"
+    test CheckNotPossibleDeselectRequiredAttributesForAccounts {
+        var requiredAttributes = "accountEntryId,emailAddress,name";
+
+        var requiredAttributesList = ListUtil.newListFromString("${requiredAttributes}");
```

Comment:
Hey @manuferreira, I think you don't need to convert the `String` to a `List` here, this will already be done by the `for` loop. But feel free to keep it if you want in your code since it's not wrong, but depending on the use of some macros, an error could happen in the type of data expected by the macro (this is not your case)